### PR TITLE
SPRXCLT-2 upgrade to Node 16

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

**Comment by nicolas2bert** [comment]
ping

**Review by miniscruff — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e wait

**Review by tcarmet — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "eslint": "^7.32.0",
     "eslint-config-airbnb": "^18.2.1",
-    "eslint-config-scality": "scality/Guidelines#71a059ad",
+    "eslint-config-scality": "scality/Guidelines#improvement/GDL-5-node-16-upgrade",
```
Comment (tcarmet) [inline package.json:26]: Use tag here?

**Comment by nicolas2bert** [comment]
@bert-e approve

**Pull request merged**